src/alto/server/estimate_throughput.py

```diff
@@ -0,0 +1,112 @@
+"""
+estimate_throughput.py
+
+A Flask app that runs a basic ALTO server.
+
+IMPORTANT NOTES:
+* This script assumes that the configuration files are located at
+	"input/g2.conf" and "output/input_routing.conf". If this is not the case,
+	then change the variables g2fp_str and infp_str
+* This script estimates RTT by summing up the delays of each link. This is

```

Comment:
Suggest changing slightly the language. Instead of:

"This is not accurate!!! But, since..."

Say something like:

"This is for now a first order approximation, since queuing and processing delay are not taken into account."